### PR TITLE
Dev plan

### PR DESCRIPTION
**Pull request by briantule**
Created the development plan as a md. Please take a look and comment any topics I have missed. Otherwise, the file can be updated for later revisions.

### CODE REVIEW

**Review by amosyu2000 — CHANGES_REQUESTED** [review]
Nice work! Please also delete the DevelopmentPlan.tex and DevelopmentPlan.pdf files.

**Review by amosyu2000 — COMMENTED** [review]
Could you also please include the issue number  in at least one of the commit messages?

File: docs/DevelopmentPlan/DevelopmentPlan.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                <td>Brian Le</th>
+                <td>Detailed steps and tools</th>
+            </tr>
+        </table>
+    </p>
+</div>
+
+# 2 Version Control
+
+Team members are are expected to use the private Github repository. Team members will also be expected to maintain repos containing clean and accurate code. Members will are required to create new branches when developing different aspects of software before merging into the main branch once functionality has been verified and is stable. Changes are highly encouraged to be made through multiple commits with a new commit made for per module or per 50 lines of code. Pull Requests are to be processed with the validation of at least two other members within the team with discussions encouraged to be held within their respective topics.
```
Comment (amoscheung99527) [inline docs/DevelopmentPlan/DevelopmentPlan.md:36]: there's an extra "are" in "Team members are are".
"Members will are required" also has an extra "are"

**Review by MannyLemos — CHANGES_REQUESTED** [review]
Under 'Roles and Responsibilities' -> 'Manny Lemos'
censor should be spelt sensor

**Review by MannyLemos — APPROVED** [review]
good

**Review by amoscheung99527 — APPROVED** [review]
nice job!

**Review by amosyu2000 — APPROVED** [review]
LGTM! Are you gonna squash merge or rebase merge?

**Comment by briantule** [comment]
Was gonna rebase merge. Wassup

**Pull request merged**